### ISSUE TITLE
Widget title for the unified mailbox is too wordy

### ISSUE THREAD

**Issue by loulouMS** [thunderbird/thunderbird-android#7792]
### Checklist

- [X] I have used the search function to see if someone else has already submitted the same feature request.
- [X] I will describe the problem with as much detail as possible.
- [X] This issue only contains a request for one single feature, **not** multiple (related) features.

### App version

6.802

### Problem you are trying to solve

'All accounts' should be simpler.

### Suggested solution

Change the title text for the unified mailbox.

**Comment by kewisch** [comment]
Can you provide a screenshot on what you're aiming for? Is this still an issue with Thunderbird for Android?

**Comment by loulouMS** [comment]
I just would like to replace the widget title by 'All accounts'. In french the title is 'Boite de réception unifiée". it's too long. Please replace it by 'All accounts' and in french by "Tous les comptes".

**Comment by kewisch** [comment]
I see what you mean, thank you. Making it work for all languages can be really difficult. Technically they are unified folders, and not necessarily all accounts. So changing it in that way wouldn't work. Is there a shorter way to express unified folders in French?

**Comment by loulouMS** [comment]
There is no shorter way but I think that the english words used are too verbose, too literary. Sometimes, a shortcut is better even it is less precise. For example, Samsung Email chooses 'All accounts'. Technically, that may be not exact but everybody understands.
Or you may choose : 'All accounts - Inbox' or 'Inbox - All accounts'

**Comment by kewisch** [comment]
Could it just be "Boîte unifiée" ? This is a bit shorter, and maybe could be considered close enough that people would understand it is an inbox instead of just a box in context?

If we move away from Unified Inbox we'll have to do this throughout the app. Not saying that is a hard blocker, but we need to consider how it is presented and how that interacts with the other components.

**Comment by loulouMS** [comment]
In french, the only word 'boîte' is not enough. Generally, we say 'Boîte de réception' or 'Courrier entrant'.
This is the reason why I proposed only 'Tous les comptes' or 'All accounts' for the widget because everybody knows it represents the inbox. n the appli, we could use 'Inbox All accounts' and 'Courrier entrant - Tous les comptes'. The words "Boite de réception unifiée' is shorter too literary. The french word 'unifié' is not very common.
Anyway, that's, of course, not very important...

**Comment by shawad2006** [comment]
<img width="1116" height="2484" alt="Image" src="https://github.com/user-attachments/assets/c7a4ac86-7e3b-4be7-9b9f-12265f56a1f8" /> Would be good if you could also choose what title shows and be able to remove it so it does not take up so much screenspace as per my example i.e. the height of the title bar re-sizes to content. I realise my 2x2 widget looks a bit odd - but it serves me well for being able to have a quick scroll to see what has come in. I know what the widget is, seeing "Unified Inbox" serves no purpose.
